playwright-tests/properties-page-tests.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { test, expect } from '@playwright/test';
+
+test('Existing property test', async ({ page }) => {
+  await page.goto('http://localhost:3000/properties/1ececec8-4bbf-445f-8de0-f563caf0bf01');
+  await expect(page.getByRole('main')).toContainText('1 Test Road');
+  await expect(page.getByRole('main')).toContainText('Test Name 1');
+});
```

Comment:
**Optional**: Consider separating into grouped tests to allow individual parts to pass or fail separately e.g.

```typescript
test.describe('details shown correctly for existing property' ...
    test('property address shown'...

    )

    test('current landlord shown'...

    )
)
```